### ISSUE TITLE
Export fails with with local path in `barrels.jungle`

### ISSUE THREAD

**Issue by bombsimon** [markw65/monkeyc-optimizer#52]
When using the VS Code plugin to add a local dependency as the `monkey.jungle` file and not a compiled project, VS Code will generate `barrels.jungle` with the following content:

```text
# Do not hand edit this file. To make changes run the
# "Configure Monkey Barrels" command.

MyBarrel = [c:\path\to\monkey.jungle]
base.barrelPath = $(base.barrelPath);$(MyBarrel )
```

With this content, exporting optimized project fails with the following message:

```text
Starting optimization step...
ERROR: SyntaxError: barrels.jungle:4,17: Expected "$(", ";", "\\", "]", [\t ], comment, quoted string, or raw string but ":" found.
```

Exporting with the default VS Code plugin option works as expected.

### Steps to reproduce

1. Clone any barrel, e.g. https://github.com/vtrifonov-esfiddle/ConnectIqDataPickers
2. Create a new project in VS Code or open an existing one

Either 
  1. Open `manifest.xml` and scroll to `Dependencies`
  2. Click `Add barrel` and navigate to and select the barrel's `monkey.jungle` file

Or
  1. Use the `Configure monkey barrel` option from the VS Code command palette.

Execute `Export optimized project`

**Comment by bombsimon** [comment]
Thanks for the swift reply! Yes, understandable!

> Meanwhile I think you can fix it by quoting the path (I realize you're not supposed to hand edit the file).

I tried both `["c:\path\to\file"]` and `"[c:\path\to\file]"` but they both fail the same way. However using a relative path like `[..\to\file]` works as expected so can confirm it's the `:`.

Let me know if there's anything I can test for you or if you want me to help find and fix the problem!

**Comment by markw65** [comment]
That makes sense. In fact I do have some tests with barrels, and I do sometimes run them on windows. But I had had to make the paths relative anyway; using the absolute paths doesn't work for different checkouts on the same machine, let alone cross-platform checkouts.

I need to look at the quoting too. `["c:\path\to\file"]` should work (paths with spaces need to be quoted, for example).

**Comment by markw65** [comment]
Fixed in v1.1.81 of the optimizer module, and 2.0.111 of the extension.

**Comment by bombsimon** [comment]
Dang, you're fast! Thank you once again for your work and contribution to the Monkey C ecosystem!
